packages/camera/camera/example/ios/RunnerTests/DeviceOrientationTests.m

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera.Test;
+@import AVFoundation;
+#import <XCTest/XCTest.h>
+
+@interface DeviceOrientationTests : XCTestCase
+
+@end
+
+@implementation DeviceOrientationTests
+
+- (void)testFLTGetUIDeviceOrientationForString {
+  XCTAssertEqual(UIDeviceOrientationPortraitUpsideDown,
+                 FLTGetUIDeviceOrientationForString(@"portraitDown"));
+  XCTAssertEqual(UIDeviceOrientationLandscapeRight,
+                 FLTGetUIDeviceOrientationForString(@"landscapeLeft"));
+  XCTAssertEqual(UIDeviceOrientationLandscapeLeft,
+                 FLTGetUIDeviceOrientationForString(@"landscapeRight"));
```

Comment:
For some reason the left/right orientation is flipped, but this seems to be intentional for some unknown reason: https://github.com/flutter/plugins/pull/3390/files#r793213256